tests/ssh_test_images/Dockerfile

```
@@ -0,0 +1,25 @@
+# Use a base image with the desired OS (e.g., Ubuntu, Debian, etc.)
+FROM ubuntu:latest
+
+# Install SSH server
+RUN apt-get update && \
+    apt-get upgrade -y
+RUN apt-get install openssh-server -y supervisor
+RUN apt-get install nano
+
+# Create an SSH user
+RUN useradd -rm -d /home/sshuser -s /bin/bash -g root -G sudo -u 1000 sshuser
```

```suggestion
RUN useradd -rm -d /home/sshuser -s /bin/bash -g root -G sudo sshuser
```
The `-u 1000` flag throws up an error saying "the UID is not unique" so we might as well just skip it as I don't think we would need it.